qa/.qa

```diff
@@ -1 +0,0 @@
-../.qa
```

Comment:
Better might be to change it to: `ln -sf . .qa`

Reply:
I was wondering about this, too. I decided against it, as

1. It's not used right now. We can create a new symlink, if the need arises. 
2. I felt it a bit awkward and to reference the local dir. 
3. It breaks with all the other `.qa` symlinks as it has a different target. 
4. I have the feeling that it might make bugs harder to find. I can imagine someone might end up in an endless loop this way. 

what do you think?

Reply:
> I was wondering about this, too. I decided against it, as
> 
>     1. It's not used right now. We can create a new symlink, if the need arises.

That is true.

>     2. I felt it a bit awkward and to reference the local dir.
>
>     3. It breaks with all the other `.qa` symlinks as it has a different target.

`suites/.qa` is different too and is `-> ../`.
 
>     4. I have the feeling that it might make bugs harder to find. I can imagine someone might end up in an endless loop this way.

You'd know pretty quick with a `ELOOP`.
 
> 
> what do you think?

Makes no difference to me which you pick :)